hibernate-core/src/main/java/org/hibernate/hql/internal/ast/HqlParser.java

```diff
@@ -348,6 +347,10 @@ private AST processIsEmpty(AST node, boolean negated) {
 	private AST createSubquery(AST node) {
 		AST ast = ASTUtil.createParent( astFactory, RANGE, "RANGE", node );
 		ast = ASTUtil.createParent( astFactory, FROM, "from", ast );
+
+		AST alias = ASTUtil.createSibling( astFactory, ALIAS, "_", node );
+		ASTUtil.insertChild( ASTUtil.createSibling( astFactory, SELECT, "select", ast ), astFactory.create(IDENT, alias.getText() ) );
+		
```

Comment:
The root cause of the bug is for 'exists' sub-query, we didn't specify 'SELECT', though we specified 'FROM'. Adding 'SELECT' fixed the issue.